tests/integration/test_wazuh_db/test_wdb_backup_configs.py

```diff
@@ -0,0 +1,233 @@
+    - Debian Jessie
+    - Debian Wheezy
+    - Red Hat 8
+    - Red Hat 7
+    - Red Hat 6
+
+references:
+    - https://documentation.wazuh.com/current/user-manual/reference/daemons/wazuh-db.html
+
+tags:
+    - wazuh_db
+'''
+
+import os
+import subprocess
+
+import pytest
+import time
+import numbers
+
+from wazuh_testing.tools.configuration import load_wazuh_configurations
+from wazuh_testing.tools.services import control_service
+from wazuh_testing.tools.file import recursive_directory_creation, remove_file, truncate_file
+from wazuh_testing.tools.monitoring import FileMonitor, generate_monitoring_callback
+from wazuh_testing.tools import LOG_FILE_PATH, WAZUH_PATH
+from wazuh_testing.modules import TIER0, WAZUH_SERVICES_STOP, WAZUH_SERVICES_START
+
+
+# Marks
+pytestmark =  [pytest.mark.linux, TIER0, pytest.mark.server]
+
+
+# Configuration
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+configurations_path = os.path.join(test_data_path, 'wazuh_db_backups_conf.yaml')
+backups_path = os.path.join(WAZUH_PATH, 'backup', 'db')
+interval = 5
+
+parameters = [{'ENABLED': 'yes', 'INTERVAL': str(interval)+'s', 'MAX_FILES':''},
+              {'ENABLED': 'yes', 'INTERVAL': str(interval)+'s', 'MAX_FILES':'value'},
+              {'ENABLED': 'yes', 'INTERVAL': '', 'MAX_FILES':'1'},
+              {'ENABLED': 'yes', 'INTERVAL': 'value', 'MAX_FILES':1},
+              {'ENABLED': 'no', 'INTERVAL': str(interval)+'s', 'MAX_FILES':1},
+              {'ENABLED': 'yes', 'INTERVAL': str(interval)+'s', 'MAX_FILES':1},
+              {'ENABLED': 'yes', 'INTERVAL': str(interval)+'s', 'MAX_FILES':3},
+              {'ENABLED': 'yes', 'INTERVAL': str(interval)+'s', 'MAX_FILES':0}
+            ]
+
+metadata = [{'ENABLED': 'yes', 'INTERVAL': str(interval)+'s', 'MAX_FILES':''},
+            {'ENABLED': 'yes', 'INTERVAL': str(interval)+'s', 'MAX_FILES':'value'},
+            {'ENABLED': 'yes', 'INTERVAL': '', 'MAX_FILES':1},
+            {'ENABLED': 'yes', 'INTERVAL': 'value', 'MAX_FILES':1},
+            {'ENABLED': 'no', 'INTERVAL': str(interval)+'s', 'MAX_FILES':1},
+            {'ENABLED': 'yes', 'INTERVAL': str(interval)+'s', 'MAX_FILES':1},
+            {'ENABLED': 'yes', 'INTERVAL': str(interval)+'s', 'MAX_FILES':3},
+            {'ENABLED': 'yes', 'INTERVAL': str(interval)+'s', 'MAX_FILES':0}
+           ]
+
+configurations = load_wazuh_configurations(configurations_path, __name__ ,
+                                           params=parameters, metadata=metadata)
+
+# Variables
+BACKUP_CREATION_CALLBACK = r'.*Created Global database backup "(backup/db/global.db-backup.*.gz)"'
+WRONG_INTERVAL_CALLBACK = r".*Invalid value for element ('interval':.*)"
+WRONG_MAX_FILES_CALLBACK = r".*Invalid value for element ('max_files':.*)"
+wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+timeout = 15
+
+
+def restart_wazuh():
+    """Restarts Wazuh."""
+    control_service(WAZUH_SERVICES_STOP)
+    control_service(WAZUH_SERVICES_START)
```

Comment:
Here you mentioned that it is not an array, and so you still don't use the one defined in conftest, however, this is not the place where it should be defined. It should be in a place where the rest can use it.